arangod/Agency/FailedLeader.cpp

```diff
@@ -238,8 +238,25 @@ bool FailedLeader::start(bool& aborts) {
     }
   }
 
+  // Exclude servers in failoverCandidates for some clone and those in Plan:
+  auto shardsLikeMe = clones(_snapshot, _database, _collection, _shard);
+  auto failoverCands = Job::findAllFailoverCandidates(
+      _snapshot, _database, shardsLikeMe);
+  std::vector<std::string> excludes;
+  for (const auto& s : VPackArrayIterator(planned)) {
+    if (s.isString()) {
+      std::string id = s.copyString();
+      if (failoverCands.find(id) == failoverCands.end()) {
```

Comment:
the comment in line 241 says we are excluding servers in failoverCandidates, but line 249 looks to me as if we would be excluding servers that are _not_ in failoverCandidates. is this intentional?